nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -319,429 +320,447 @@ def _threshold_and_rescale(data, threshold, vmin, vmax):
-                # so that background map becomes visible
-                surf_map_face_colors[kept_indices, 3] = 0.7
-
-            face_colors = mix_colormaps(surf_map_face_colors, bg_face_colors)
-
-            if colorbar:
-                cbar_vmin = cbar_vmin if cbar_vmin is not None else vmin
-                cbar_vmax = cbar_vmax if cbar_vmax is not None else vmax
-                ticks = _get_ticks(
-                    cbar_vmin, cbar_vmax, cbar_tick_format, threshold
-                )
-                our_cmap, norm = _get_cmap(
-                    cmap, vmin, vmax, cbar_tick_format, threshold
-                )
-                bounds = np.linspace(cbar_vmin, cbar_vmax, our_cmap.N)
-
-                # we need to create a proxy mappable
-                proxy_mappable = ScalarMappable(cmap=our_cmap, norm=norm)
-                proxy_mappable.set_array(surf_map_faces)
-                figure._colorbar_ax, _ = make_axes(
-                    axes,
-                    location="right",
-                    fraction=0.15,
-                    shrink=0.5,
-                    pad=0.0,
-                    aspect=10.0,
-                )
-                figure._cbar = figure.colorbar(
-                    proxy_mappable,
-                    cax=figure._colorbar_ax,
-                    ticks=ticks,
-                    boundaries=bounds,
-                    spacing="proportional",
-                    format=cbar_tick_format,
-                    orientation="vertical",
-                )
-
-            # fix floating point bug causing highest to sometimes surpass 1
-            # (for example 1.0000000000000002)
-            face_colors[face_colors > 1] = 1
-
-            p3dcollec.set_facecolors(face_colors)
-            p3dcollec.set_edgecolors(face_colors)
-
-        if title is not None:
-            axes.set_title(title)
-
-        return save_figure_if_needed(figure, output_file)
-
-    def _plot_surf_contours(
-        self,
-        surf_mesh=None,
-        roi_map=None,
-        hemi=DEFAULT_HEMI,
-        levels=None,
-        labels=None,
-        colors=None,
-        legend=False,
-        cmap="tab20",
-        title=None,
-        output_file=None,
-        axes=None,
-        figure=None,
-        **kwargs,
+        # fix floating point bug causing highest to sometimes surpass 1
+        # (for example 1.0000000000000002)
+        face_colors[face_colors > 1] = 1
+
+        p3dcollec.set_facecolors(face_colors)
+        p3dcollec.set_edgecolors(face_colors)
+
+    if title is not None:
+        axes.set_title(title)
+
+    return save_figure_if_needed(figure, output_file)
+
+
+def _plot_surf_contours(
+    surf_mesh=None,
+    roi_map=None,
+    hemi=DEFAULT_HEMI,
+    levels=None,
+    labels=None,
+    colors=None,
+    legend=False,
+    cmap="tab20",
+    title=None,
+    output_file=None,
+    axes=None,
+    figure=None,
+    **kwargs,
+):
```

Comment:
For private functions, I still like to have a one-line docstring saying what it does.